stopcovid/dialog/engine.py

```diff
@@ -304,7 +304,7 @@ def _check_response(
     def _drill_requested(
         self, dialog_state: DialogState, base_args: Dict[str, Any]
     ) -> Optional[List[DialogEvent]]:
-        if not dialog_state.current_drill or self._current_drill_is_stale(dialog_state):
+        if not dialog_state.current_drill:  # or self._current_drill_is_stale(dialog_state):
```

Comment:
stop the bleeding while I'm working on registration.

Once registration is fixed we can turn this back on.